includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -485,22 +485,23 @@ public function output_payments_settings_screen() {
 	/**
 	 * Generates the configuration values, needed for payment fields.
 	 *
-	 * Isolated as a separate method in order to be avaiable both
+	 * Isolated as a separate method in order to be available both
 	 * during the classic checkout, as well as the checkout block.
 	 *
 	 * @return array
 	 */
 	public function get_payment_fields_js_config() {
 		return [
-			'publishableKey'         => $this->account->get_publishable_key( $this->is_in_test_mode() ),
-			'accountId'              => $this->account->get_stripe_account_id(),
-			'ajaxUrl'                => admin_url( 'admin-ajax.php' ),
-			'createSetupIntentNonce' => wp_create_nonce( 'wcpay_create_setup_intent_nonce' ),
-			'genericErrorMessage'    => __( 'There was a problem processing the payment. Please check your email inbox and refresh the page to try again.', 'woocommerce-payments' ),
-			'fraudServices'          => $this->account->get_fraud_services_config(),
-			'features'               => $this->supports,
-			'forceNetworkSavedCards' => WC_Payments::is_network_saved_cards_enabled(),
-			'locale'                 => WC_Payments_Utils::convert_to_stripe_locale( get_locale() ),
+			'publishableKey'           => $this->account->get_publishable_key( $this->is_in_test_mode() ),
+			'accountId'                => $this->account->get_stripe_account_id(),
+			'ajaxUrl'                  => admin_url( 'admin-ajax.php' ),
+			'createSetupIntentNonce'   => wp_create_nonce( 'wcpay_create_setup_intent_nonce' ),
+			'createPaymentIntentNonce' => wp_create_nonce( 'wcpay_create_payment_intent_nonce' ),
```

Comment:
Is this used anywhere?

Reply:
This actually was a relic from when I was creating the payment via AJAX, so it shouldn't have been here...But then after the conversation I had with @bborman22 above, we decided to switch the payment intent creation back to AJAX, so now I do need it again after all! (This nonce now gets checked over [here](https://github.com/Automattic/woocommerce-payments/blob/1e748f957dd221d66cf40aec2b83a9d7ea310fee/includes/payment-methods/class-upe-payment-gateway.php#L64).)